docs/src/content/tutorial/setting_and_running_hydrus.md

```diff
@@ -0,0 +1,145 @@
+---
+name: Setting up and Running Hydrus
+menu: Tutorial
+---
+
+# Setting Up and Running hydrus
+
+> This tutorial assumes you have a basic understanding of python programming language, Hydra, API documentation created using Hydra, Linked Data, web servers, and REST APIs.
```

Comment:
python -> Python
(this change can be made everywhere applicable in this document)